python_modules/dagster/dagster/_core/definitions/partitioned_schedule.py

```diff
@@ -69,6 +72,8 @@ def build_schedule_from_partitioned_job(
     day_of_week: Optional[int] = None,
     day_of_month: Optional[int] = None,
     default_status: DefaultScheduleStatus = DefaultScheduleStatus.STOPPED,
+    cron_schedule: Optional[str] = None,
```

Comment:
can we assert that `cron_schedule` must be none if `minute_of_hour`/`hour_of_day`/`day_of_week`/`day_of_month` are provided and vice versa?

To prevent users from passing in unused fields